### PR TITLE
common: Remove redundant includes

### PR DESCRIPTION
**Pull request by badone**
3rd in a series

Fixes: http://tracker.ceph.com/issues/19883 (Partially)

Signed-off-by: Brad Hubbard <bhubbard@redhat.com>

### CODE REVIEW

File: src/cls/lua/cls_lua_client.h [hunk 1]
```diff
@@ -1,8 +1,12 @@
 #ifndef CLS_LUA_CLIENT_HPP
 #define CLS_LUA_CLIENT_HPP
 #include <string>
-#include <vector>
-#include "include/rados/librados.hpp"
+
+#include "include/buffer_fwd.h"  // for bufferlist
+
+namespace librados {
+    class IoCtx;
```
Comment (tchaikov) [inline src/cls/lua/cls_lua_client.h]: wrong indent.

Comment (badone) [inline src/cls/lua/cls_lua_client.h]: @tchaikov good eye for detail my friend. I blame the missing modelines =D

**Comment by badone** [comment]
Note to self. #15086 is waiting on this PR to merge.

**Comment by yuriw** [comment]
http://pulpito.ceph.com/yuriw-2017-05-15_22:59:10-rados-wip-yuri-testing_2017_5_16-distro-basic-smithi/

http://pulpito.ceph.com/yuriw-2017-05-16_13:48:34-rados-wip-yuri-testing_2017_5_16---basic-smithi/

**Pull request merged**